yxi/axi-calyx/axi-combined-calyx.futil

```diff
@@ -673,7 +572,7 @@ component m_write_channel(
 
         //set to 1 after valid has been high even once
         wvalid_was_high.in = 1'b1;
-        wvalid_was_high.write_en = !(wvalid.out & WREADY) & !wvalid_was_high.out ? 1'b1;
+        wvalid_was_high.write_en = !(wvalid.out & WREADY & !wvalid_was_high.out) ? 1'b1;
```

Comment:
This partitions assignment to `wvalid.write_en` whereas before there was some uncovered component of the guard (see ~5 lines up)